index.css

```diff
@@ -13,3 +13,9 @@ main {
 #reset {
   margin: 20px;
 }
+.message, #reset {
+  display: none;
+}
+.message, #reset {
+  display: block;
+}
```

Comment:
The second rule just overrides the first; Also the reset button just isn't being shown, either at the start of the game, or at completion.